dev-tools/download_windows_deps.ps1

```diff
@@ -9,7 +9,7 @@
 # limitation.
 #
 $ErrorActionPreference="Stop"
-$Archive="usr-x86_64-windows-2016-9.zip"
+$Archive="usr-x86_64-windows-2016-10.zip"
 $Destination="C:\"
 # If PyTorch is not version 1.13.1 then we need the latest download
```

Comment:
The condition here needs to be changed to look for a file that only exists in Boost 1.83.

https://github.com/elastic/ml-cpp/pull/638/files#diff-0a356335cbaadf7c9897e08805d11e4d124df92f6b166867982739296cd008b1 is an example from a previous upgrade.

(It's much easier to detect if latest Boost is present than to detect latest PyTorch.)